src/Symfony/Component/OptionsResolver/Tests/OptionsResolver2Dot6Test.php

```diff
@@ -1557,4 +1522,57 @@ public function testCountFailsOutsideResolve()
 
         count($this->resolver);
     }
+
+    /**
+     * @dataProvider getFormatValueData
+     * @expectedException \Symfony\Component\OptionsResolver\Exception\InvalidOptionsException
+     */
+    public function testFormatValue($actualType, $allowedType, $exceptionMessage)
```

Comment:
please name this test `testResolveFailsIfInvalidType` and at the same lines of the old tests. this makes merging easier.